programs/pluto/state.c

```diff
@@ -1798,7 +1828,25 @@ static bool v2_spi_predicate(struct state *st, void *context)
 		}
 #endif
 	}
-	return false;
+	return ret;
+}
+
+struct child_sa *find_v2_child_sa_by_spi(ipsec_spi_t spi, int8_t protoid,
```

Comment:
this function already exists in main, so i dont think this can merge as is ?

Reply:
that PR does not merge conflict. I don't see it you are possibly confusing it.